frontend/src/services/GooglePlacesService.ts

```diff
@@ -1,33 +1,34 @@
-import axios, { AxiosError, AxiosResponse } from 'axios';
-import * as dotenv from 'dotenv';
+import axios, { AxiosError, AxiosResponse } from "axios";
+import * as dotenv from "dotenv";
 
 /* eslint-disable  no-console */
 // todo: remove this eslint-disable
 class GooglePlacesService {
+  public currRestaurants: Map<string, Record<string, unknown>>;
```

Comment:
Just a note- I had to use Record instead of object because the linter complained!

Reply:
Just curious, what's the `Record<string, unknown>` for?

Reply:
Like would we want the value to be an interface we define as `RestaurantDetails`?

Reply:
I'm using the Record to manage the current state for the user- so all the restaurants they've clicked on or have looked at so far! We could have an interface- I didn't think of this but it might actually be easier connecting with the restaurant cards

Reply:
Ya I like the idea of having an interface! I've started defining some under `src/types/` so if you wanted to add one for `RestaurantDetails` in there that would be great!